tests/scripts/abi_clean.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# # # This source code is protected under the license referenced at
+# # # https://github.com/NRLMMD-GEOIPS.
+
+geoips run single_source $GEOIPS_TESTDATA_DIR/test_data_geocolor/data/goes16/20240924.1500/* \
+    --reader_name abi_netcdf \
+    --product_name GeoColor \
+    --compare_path "$GEOIPS_PACKAGES_DIR/geocolor/tests/outputs/abi_clean/<product>_image" \
+    --reader_kwargs '{"self_register": "LOW"}' \
+    --self_register_dataset 'Full-Disk' \
+    --self_register_source abi \
+    --output_formatter unprojected_image \
+    --output_formatter_kwargs '{"x_size": "10848", "y_size": "10848"}' \
```

Comment:
We might want to at least make this full disk test smaller ?  2048x2048 ?

Reply:
Or are these temporary and will not use full disk unprojected for the self register test ?